### ISSUE TITLE
Возможно ли использовать этот проект на arm архитектуре?

### ISSUE THREAD

**Issue by seniorivn** [yandex/tomita-parser#42]
судя по бинарникам в [релизе](https://github.com/yandex/tomita-parser/releases/tag/v1.0) mystem существует только под x86_64  платформу, значит ли это, что под арм использование не доступно?

**Comment by vbocharov** [comment]
Да, для использования на арме потребуется другая морфология. Возможно, что придётся изменить что-то в скриптах cmake.
